src/calendar/stories/RangeCalendar.stories.tsx

```diff
@@ -1,6 +1,14 @@
 import * as React from "react";
 import { Meta } from "@storybook/react";
 import { addDays, addWeeks, setDate, subDays, subWeeks } from "date-fns";
+import "./range-style.css";
+
+import {
+  ChevronLeft,
+  ChevronRight,
+  DoubleChevronLeft,
+  DoubleChevronRight,
+} from "./svg-icons";
```

Comment:
arrange the imports

Reply:
Don't want do do that, that css file looks really wierd inbetween all those imports. i will put them on very top of the file for each storybook.

Reply:
😆 Ok, thats no issue